docs/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc.rst

```diff
@@ -160,7 +160,9 @@ Add New DC
 #. When all nodes are up and running ``ALTER`` the following Keyspaces in the new nodes:
 
    * Keyspace created by the user (which needed to replicate to the new DC).
-   * System: ``system_auth``, ``system_distributed``, ``system_traces`` For example, replicate the data to three nodes in the new DC.
+   * System: ``system_distributed``, ``system_traces``, for example, replicate the data to three nodes in the new DC.
+
+   .. scylladb_include_flag:: system-auth-alter-info.rst
```

Comment:
so this info is duplicated here and in below in "After Upgrading from 5.4", but maybe that's good ?

Reply:
it's ok for me